src/app/modules/verification/verification.service.ts

```diff
@@ -322,7 +324,13 @@ export const sendNewOtp = ({
           return errAsync(new WaitForOtpError())
         }
 
-        return sendOtpForField(transaction.formId, field, recipient, otp)
+        return sendOtpForField(
+          transaction.formId,
+          fieldId,
+          field,
```

Comment:
Do we need to pass in the fieldId? Won't the fieldId be present in given field parameter already?

Reply:
thanks for highlighting, edited accordingly